### PR TITLE
Update README.md to match with new repo

### PR DESCRIPTION
**Pull request by skyyt15**
replace all `Exiled-Official` with `ExMod-Team`

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
   <div>
     <img src="https://flagsapi.com/US/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/README.md">English</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/README.md">English</a>
```
Comment (louis1706) [inline .github/README.md:33]:
```suggestion
    <a href=".github/documentation/README.md">English</a>
```

File: .github/README.md [hunk 2]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
...
   </div>
   <br>
   <div>
     <img src="https://flagsapi.com/RU/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/localization/README-Русский.md">Русский</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/localization/README-Русский.md">Русский</a>
```
Comment (louis1706) [inline .github/README.md:39]:
```suggestion
    <a href=".github/documentation/localization/README-Русский.md">Русский</a>
```

File: .github/README.md [hunk 3]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
...
   </div>
   <br>
   <div>
     <img src="https://flagsapi.com/CN/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/localization/README-中文.md">中文</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/localization/README-中文.md">中文</a>
```
Comment (louis1706) [inline .github/README.md:45]:
```suggestion
    <a href=".github/documentation/localization/README-中文.md">中文</a>
```

File: .github/README.md [hunk 4]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
...
   </div>
   <br>
   <div>
     <img src="https://flagsapi.com/ES/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/localization/README-ES.md">Español</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/localization/README-ES.md">Español</a>
```
Comment (louis1706) [inline .github/README.md:51]:
```suggestion
    <a href=".github/documentation/localization/README-ES.md">Español</a>
```

File: .github/README.md [hunk 5]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
...
   </div>
   <br>
   <div>
     <img src="https://flagsapi.com/PL/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/localization/README-PL.md">Polski</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/localization/README-PL.md">Polski</a>
```
Comment (louis1706) [inline .github/README.md:57]:
```suggestion
    <a href=".github/documentation/localization/README-PL.md">Polski</a>
```

File: .github/README.md [hunk 6]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
...
   </div>
   <br>
   <div>
     <img src="https://flagsapi.com/BR/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/localization/README-BR.md">Português-BR</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/localization/README-BR.md">Português-BR</a>
```
Comment (louis1706) [inline .github/README.md:63]:
```suggestion
    <a href=".github/documentation/localization/README-BR.md">Português-BR</a>
```

File: .github/README.md [hunk 7]
```diff
@@ -30,85 +30,85 @@ Localized READMEs
...
   </div>
   <br>
   <div>
     <img src="https://flagsapi.com/IT/flat/64.png" height=30>
     <br>
-    <a href="https://github.com/Exiled-Official/EXILED/blob/master/.github/documentation/localization/README-IT.md">Italiano</a>
+    <a href="https://github.com/ExMod-Team/EXILED/blob/master/.github/documentation/localization/README-IT.md">Italiano</a>
```
Comment (louis1706) [inline .github/README.md:69]:
```suggestion
    <a href=".github/documentation/localization/README-IT.md">Italiano</a>
```

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Comment by louis1706** [comment]
need update on Localised file

**Comment by Misaka-ZeroTwo** [comment]
> need update on Localised file

That could be done in a separate PR. Plus not super important.

**Comment by louis1706** [comment]
ok

**Review by louis1706 — APPROVED** [review]

**Pull request merged**